templates/index.html

```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com">
```

Comment:
```suggestion
                <a class="sponsors__link" href="https://vercel.com?utm_source=[team-name]&utm_campaign=oss">
```

Per https://vercel.com/support/articles/can-vercel-sponsor-my-open-source-project